dropfile_formats/CALLINFO.BBS

```diff
@@ -0,0 +1,36 @@
+{UR}
```

Comment:
Can we call these "dropfile_templates" to match the 'template' naming elsewhere?

Reply:
Will do